File: micro-service/infra/src/main/java/tao/berich/microservice/infra/fund/MongoConfig.java
package tao.berich.microservice.infra.fund;

import org.springframework.context.annotation.Configuration;
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;

@Configuration
@EnableMongoRepositories(basePackages = "tao.berich.microservice.infra.fund.repository")
public class MongoConfig {
}

File: identity-manager/infra/src/main/kotlin/tao/identitymanager/infra/common/MongoConfig.kt
package tao.identitymanager.infra.common

import org.springframework.context.annotation.Configuration
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories

@Configuration
@EnableMongoRepositories(basePackages = ["tao.identitymanager.infra"])
class MongoConfig {

/*    override fun getDatabaseName(): String {
        return "berich-dev"
    }*/
}